### PR TITLE
Parameterize conveyer

### PR DESCRIPTION
**Pull request by sam-falvo**

### CODE REVIEW

File: conveyer/conveyer.py [hunk 1]
```diff
@@ -117,38 +118,52 @@ def Conveyer(config, file_override=None, renamer=None):
...
-
-@app.route('/rotate', methods=['POST'])
-def rotate_log(request):
-    """Rotate the log, and report the filename back to the client."""
-    fn = conveyer.rotate_logs()
-    request.response = 200
-    return fn
+    def reset(self):
+        """Obtains a Conveyer instance to forward requests to."""
+        assert False
```
Comment (cyli) [inline conveyer/conveyer.py]: Does anything call this function yet?  It seems like it'd just raise an exception, and resetting the Conveyer will never actually happen?

**Comment by cyli** [comment]
Really excellent documentation!  :D  Extremely thorough - I wonder if it would be useful to provide an example logstash config, though?  (Something simple, for example, something you might use to test this?)

File: conveyer/conveyer.py (hunk 1 above)
Comment (sam-falvo) [inline conveyer/conveyer.py]: Good catch.  It's dead code, and I forgot to remove it.

**Comment by sam-falvo** [comment]
Done and done.  Can you re-review?  Thanks!

**Comment by cyli** [comment]
LGTM!  :)

**Comment by sam-falvo** [comment]
Fixes #5

**Comment by sam-falvo** [comment]
Fixes #6

**Comment by sam-falvo** [comment]
Fixes #3

**Pull request merged**